sof_wrapper/wrapped_session.py

```diff
@@ -0,0 +1,47 @@
+import pickle
+from flask import g, current_app, session
+
+
+def get_session_value(key, default=None):
+    """Return session value for given key
+
+    Typically flask-session allows for direct access via `session.get`
+    but that only works when a browser cookie is available, which is
+    not the case when launched from the dashboard.
+
+    Until resolved, this function tries local and configured session
+    and returns a value if found.
+    """
+    if key in session:
+        return session.get(key, default)
+
+    # session_id stored on entry point in `fhir_router`
+    if 'session_id' in g:
+        return get_redis_session_data(g.session_id).get(key, default)
+
+
+def set_session_value(key, value):
+    if 'session_id' not in g:
+        session[key] = value
+        return
+
+    session_data = get_redis_session_data(g.session_id)
+    session_data[key] = value
```

Comment:
Does this actually set `key` persistently? `get_redis_session_data()` doesn't return an actual session that will persist changes

Reply:
updated to raise if we attempt to set a session variable w/o a session cookie.  i.e. catch the misleading no-op